### DISCUSSION TITLE
推荐使用 electron-ipc-cat 做 ipc

### DISCUSSION THREAD

**Discussion by linonetwo** [Future-Scholars/paperlib#587] [Development]
刚看到 https://paperlib.app/cn/extension-doc/#%E8%BF%9B%E7%A8%8B%E9%97%B4%E9%80%9A%E4%BF%A1 ，推荐用 https://github.com/linonetwo/electron-ipc-cat 就不用自己维护了。

在 https://github.com/tiddly-gittly/TidGi-Desktop 里已经使用多年。

**Comment by GeoffreyChen777** [comment]
paperlib 不只有main renderer。还有 utilityProcess。跨process不仅需要传递信息。我们还实现了跨process event监听等。

**Reply by linonetwo** [reply]
我是用 rxjs 的 Observable 来实现监听，不过第三个进程应该没适配，那就不能只用 ipcMain 和 ipcRenderer 了

**Reply by GeoffreyChen777** [reply]
这部分属于Paperlib比较核心的东西。因此还是不使用第三方库。自己维护比较好。
